source/scpd/types/Stellar_types.d

```diff
@@ -68,6 +72,23 @@ struct PublicKey {
     alias ed25519_ this;
 }
 
+package(scpd)
+struct UTXOHash {
+    extern(D) this(uint512 value) @safe pure nothrow @nogc
+    {
+        this.utxo_hash_ = value;
+    }
+
+    extern(D) this(typeof(this.tupleof) args) @safe pure nothrow @nogc
+    {
+        this.tupleof = args;
+    }
+
+    int32_t type_;
+    uint512 utxo_hash_;
+    alias utxo_hash_ this;
+}
```

Comment:
I think this should be handled one level higher, e.g. the `PublicKey` type has to go.